package.json

```diff
@@ -11,7 +11,53 @@
     "start": "npm-run-all -p -c server sass:watch",
     "server": "ws --directory test/",
     "sass:compile": "node-sass test/style.scss test/style.css",
-    "sass:watch": "npm-run-all sass:compile \"sass:compile -- -w\""
+    "sass:watch": "npm-run-all sass:compile \"sass:compile -- -w\"",
+    "release": "standard-version -t ''"
+  },
+  "standard-version": {
```

Comment:
Add the config into a dedicated style (separation of concerns) to not bloat the `package.json`